apm-agent-core/src/main/java/co/elastic/apm/agent/report/serialize/DslJsonSerializer.java

```diff
@@ -715,6 +715,11 @@ private void serializeContext(final TransactionContext context, TraceContext tra
         }
         serializeRequest(context.getRequest());
         serializeResponse(context.getResponse());
+        if (context.hasCustom()) {
+            writeFieldName("custom");
+            serializeLabels(context.getCustomIterator(), replaceBuilder, jw);
```

Comment:
Maybe change the method name to `serializeDictionary` or something of that sort instead?